MOBILE_API.xml

```diff
@@ -584,47 +573,47 @@
     
     <enum name="TextFieldName">
         <element name="mainField1">
-            <description>The first line of first set of main fields of the persistent display; applies to "Show"</description>
```

Comment:
I don't think we should remove this note. Maybe change the wording to something like: "Currently used in: Show". Same goes for all context removals for this enum